src/ctapipe_io_nectarcam/__init__.py

```diff
@@ -433,6 +435,15 @@ class NectarCAMEventSource(EventSource):
         help='If True, flat field and pedestal events are also calibrated.'
     ).tag(config=True)
 
+    def _correct_tel_id(self,tel_id,run):
```

Comment:
Add comments to explain what is done here? If I had to modify the code in the future I would not be able to understand it.

Reply:
Yes. I've now added some comments in the code about that.